### ISSUE TITLE
Looking forward to the 11 layer model with 93.2% accuracy.

### ISSUE THREAD

**Issue by ruc98** [Yochengliu/DensePoint#3]
.

**Comment by sheshap** [comment]
I am able to reproduce Acc: 0.931929 which is 93.2 as claimed in the paper.

**Comment by ruc98** [comment]
That great! Did you change any training configurations in `config_cls.yaml`? I tried to implement the exact 11 layer architecture mentioned in the paper with the same config settings but could not get the required accuracy.

**Comment by sheshap** [comment]
@ruc98 I didn't change anything. From where did you see 11 layers?. Classification network has 14 layers and segmentation has 23 layers right?

**Comment by ruc98** [comment]
@sheshappanavar Yes, I mean L=11 excluding the 3 fc layers. So are you able to reproduce 93.2% using the given L=6 code or did you implement the L=11 code given in the paper? Also without the voting evaluation what is your accuracy?

Thanks

**Comment by sheshap** [comment]
@ruc98 The 93.2% which I got is using the given L=6 code, with voting.
I have not checked without voting. Keep us posted if you try L=11 with or without voting.
